test/addons-napi/test_reference/test.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+// Flags: --expose-gc
+
+const common = require('../../common');
+const assert = require('assert');
+
+const test_reference = require(`./build/${common.buildType}/test_reference`);
+
+// This test script uses external values with finalizer callbacks
+// in order to track when values get garbage-collected. Each invocation
+// of a finalizer callback increments the finalizeCount property.
+assert.strictEqual(0, test_reference.finalizeCount);
```

Comment:
Can you change the order of the arguments to `actual, expected` in `strictEqual()` throughout the test.